internal/services/kusto/kusto_iothub_data_connection_resource.go

```diff
@@ -132,6 +132,13 @@ func resourceKustoIotHubDataConnection() *pluginsdk.Resource {
 					ValidateFunc: validation.StringIsNotEmpty,
 				},
 			},
+
+			"retrieval_start_date": {
+				Type:         pluginsdk.TypeString,
+				Optional:     true,
+				ForceNew:     true,
```

Comment:
It looks like this property is updatable

Reply:
Thank you for your feedback. I will test this update locally and then work on making the argument updatable once I have completed the testing.